src/components/nav-bar/nav-bar.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 1em;
+  transition: all 1s ease-in-out;
+}
```

Comment:
Components styles should not affect anything outside of it self. 
e.g. if we use this on Oberd core, it will affect the whole application.

Reply:
will remove